StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheet+APITest.swift

```diff
@@ -65,6 +65,28 @@ class PaymentSheetAPITest: STPNetworkStubbingTestCase {
         return newCardPaymentOption
     }()
 
+    lazy var newCardDefaultPaymentOption: PaymentSheet.PaymentOption = {
```

Comment:
Any way we can DRY up the duplication of code between this and "newCardPaymentOption"?